industrial_taxonomy/pipeline/official/population/flow.py

```diff
@@ -0,0 +1,51 @@
+# Flow to fetch population data
+
+from metaflow import FlowSpec, project, step
+from metaflow.decorators import StepDecorator
+
+URL = (
+    "https://www.ons.gov.uk/file?uri=/peoplepopulationandcommunity/populationandmigration/populationestimates/",
+    "datasets/populationestimatesforukenglandandwalesscotlandandnorthernireland/",
+    "mid2020/ukpopestimatesmid2020on2021geography.xls",
```

Comment:
If the comma separation from these is removed then no need for `self.url = "".join(URL)` below